packages/create-sitecore-jss/scripts/samples.json

```diff
@@ -0,0 +1,66 @@
+﻿[
+  {
+    "initializers": [
+      "nextjs"
+    ],
+    "args": {
+      "appName": "nextjs-vanilla",
+      "destination": "..\\..\\samples\\nextjs-vanilla",
+      "fetchWith": "GraphQL",
+      "prerender": "SSR",
```

Comment:
Do we have `SSG` by default?

Reply:
We can add SSG samples later if needed, when nextjs app build step can be added. For now it does linting only.